<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>411</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Trainer</string>
     </attribute>
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>lastname_tr</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>idTreiner</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_4"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_5"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>firstname_tr</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>year_of_birth_tr</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>rank</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>220</y>
        <width>243</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>RenewTreiner</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>DeleteTreiner</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>AddTreiner</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_19">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>prev</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_20">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>280</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>next</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Sportsmen</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>260</y>
        <width>255</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>RenewTreiner</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>AddSportsmen</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string>DeleteTreiner</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>248</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>idSportsmen</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>lastname_sp</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>firstname_sp</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>role</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>year_of_birth_sp</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>height</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>weight</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>date_hw</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>number</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_6">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_7"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_8"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_9"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_10"/>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEdit_11"/>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEdit_12"/>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="lineEdit_13"/>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="lineEdit_14"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_21">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>310</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>prev</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_22">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>310</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>next</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Sponsor</string>
     </attribute>
     <widget class="QWidget" name="formLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_6">
       <item row="0" column="0">
        <widget class="QLabel" name="label_34">
         <property name="text">
          <string>id_Sponsor</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_33">
         <property name="text">
          <string>name_sponsor</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_33">
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_34"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>259</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_16">
         <property name="text">
          <string>AddSponsor</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_17">
         <property name="text">
          <string>RenewSponsor</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_18">
         <property name="text">
          <string>DeleteSponsor</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_23">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>prev</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_24">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>130</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>next</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Transport</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>275</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_13">
         <property name="text">
          <string>AddTransport</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_14">
         <property name="text">
          <string>RenewTransport</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_15">
         <property name="text">
          <string>DeleteTransport</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="label_30">
         <property name="text">
          <string>idTransport</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_31">
         <property name="text">
          <string>type</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_32">
         <property name="text">
          <string>capacity_tr</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_30">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_31"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_32"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_25">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>prev</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_26">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>160</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>next</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Club</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>260</y>
        <width>239</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_11">
         <property name="text">
          <string>Change</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_10">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_12">
         <property name="text">
          <string>Del</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>248</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Sponsor</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>Transport</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>Treiner</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>Sportsmen</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>Gum</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>idClub</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>nameClub</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>year_create</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>city</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_21"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_22"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_23"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_24"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_25"/>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEdit_26">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEdit_27"/>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="lineEdit_28"/>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="lineEdit_29"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_27">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>310</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>prev</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_28">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>310</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>next</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Gum</string>
     </attribute>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>187</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>idGum</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>name_gum</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>address</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>capacity</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>phone_number</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>category</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_15"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_16">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_17"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_18"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_19"/>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEdit_20"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>210</y>
        <width>239</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string>RenewGum</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_8">
         <property name="text">
          <string>AddGum</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_9">
         <property name="text">
          <string>DeleteGum</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_29">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>280</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>prev</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_30">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>280</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>next</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
